webpack.common.js

```diff
@@ -43,6 +47,80 @@ module.exports = {
   },
   plugins: [
     new CleanWebpackPlugin(),
+    new CopyPlugin([
+      // fonts
+      {
+        context: './node_modules/@fortawesome/fontawesome-free/css',
+        from: 'all.min.css',
+        to: resolve(vendor, 'fontawesome', 'css')
+      },
+      {
+        context: './node_modules/@fortawesome/fontawesome-free',
+        from: 'webfonts',
+        to: resolve(vendor, 'fontawesome', 'webfonts')
+      },
+      {
+        context: './node_modules/@fortawesome/fontawesome-free',
+        from: 'LICENSE.txt',
+        to: resolve(vendor, 'fontawesome')
+      },
+      {
+        context: './node_modules/@openfonts/open-sans_all',
+        from: 'files/*-400*',
+        to: resolve(vendor, 'open-sans_all')
+      },
+      {
+        context: './node_modules/@openfonts/open-sans_all',
+        from: 'index.css',
+        to: resolve(vendor, 'open-sans_all')
+      },
+      {
+        context: './node_modules/@openfonts/open-sans_all',
+        from: 'LICENSE.md',
+        to: resolve(vendor, 'open-sans_all')
+      },
+      {
+        context: './node_modules/@openfonts/lato_latin-ext',
+        from: 'files/*-400*',
```

Comment:
Shouldn't we include all files / weights?

Reply:
Was just working off what was being loaded before...